hack/jenkins/common.sh

```diff
@@ -31,7 +31,8 @@ export KUBECONFIG="${TEST_HOME}/kubeconfig"
 export PATH=$PATH:"/usr/local/bin/:/usr/local/go/bin/:$GOPATH/bin"
 
 # installing golang so we could do go get for gopogh
-sudo ./installers/check_install_golang.sh "1.13.4" "/usr/local" || true
+WANT_GOLANG_VERSION=$(grep '^GO_VERSION' Makefile | awk '{ print $3 }')
```

Comment:
This line doesn't work in jenkins, in Jenkins job configurations we do not copy the Makefile to the agent that runs the e2e (only the e2e and the scripts)

Reply:
Thanks :100: